.eslintrc

```diff
@@ -45,5 +45,8 @@
   },
   "plugins": [
     "react"
-  ]
+  ],
+  "globals": {
+        OPIDC_HOST: true
```

Comment:
I pulled the name from: https://github.com/CDCgov/SDP-Vocabulary-Service/blob/master/config/settings.yml#L14 In fact I frequently mistyped it as OIDC because that seemed like a more sane name to me.